client/src/components/dashboard/Poll.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Card from "./Card";
+import "./Poll.css";
+import { BsChatLeftHeart } from "react-icons/bs";
+
+const Poll = ({ pollId }) => {
```

Comment:
Is the poll supposed to have functionality or is that coming later?

Reply:
discussed